components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.hpp

```diff
@@ -77,9 +77,6 @@ class MAMMicrophysics final : public scream::AtmosphereProcess {
   void run_impl(const double dt) override;
   void finalize_impl() override;
 
-  // performs some checks on the tracers group
-  void set_computed_group_impl(const FieldGroup& group) override;
```

Comment:
Unrelated, but this process doesn't used the bundled tracers, so no need to set the group.